jekyll/_config.yml

```
@@ -23,6 +23,7 @@ baseurl: "" # the subpath of your site, e.g. /blog
 url: "" # the base hostname & protocol for your site, e.g. http://example.com
 twitter_username: jekyllrb
 github_username:  jekyll
+destination: public
```

```suggestion
destination: public
permalink: pretty
```

Let's use pretty permalinks so that it doesn't show .html suffix